solidity/dashboard/src/pages/liquidity/index.js

```diff
@@ -6,7 +6,6 @@ LiquidityPage.route = {
   headerTitle: "Liquidity Rewards",
   path: "/liquidity",
   exact: true,
-  withConnectWalletGuard: true,
   emptyStateComponent: EmptyStatePage,
```

Comment:
I think we can remove `emptyStateComponent` field as well. Also, we can remove `EmptyStatePage` file for the `LiqidityPage`.